101-machine-learning-service-create/metadata.json

```diff
@@ -0,0 +1,19 @@
+{
+  "$schema": "https://aka.ms/azure-quickstart-templates-metadata-schema#",
+  "type": "QuickStart",
+  "validationType": "Manual",
```

Comment:
why does this require manual validation?

Reply:
inputs for modelIds and/or models are very very specific to the user's machine learning models and we cant provide a default value.

Reply:
We'll need to be able to validate it one way or another - so if we can't automate it we'll need instructions on how to do it manually.